ietf/templates/group/active_wgs.html

```diff
@@ -31,8 +31,9 @@ <h3 class="mt-3 navskip">
                 {% endfor %}
             </ul>
         {% endif %}
+        {% comment %}
```

Comment:
Why comment instead of delete?

Reply:
I thought maybe we bring them back in the future?

Reply:
Unless you mean "in the next week or so", I'd rather delete it - recreating something like it when/if we want it back won't be that hard.